src/shipit_dashboard/shipit_bot/cli.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+from shipit_bot.sync import BotRemote
+
+
+def main():
```

Comment:
would it be possible to use [click](http://click.pocoo.org) instead of ``argparse``. to use similar tools along projects.